helm/app-exporter/templates/configmap.yaml

```diff
@@ -10,25 +10,18 @@ data:
     server:
       enable:
         debug:
-          server: true
+          server: {{ .Values.config.debug }}
       listen:
-        address: 'http://0.0.0.0:8000'
+        address: http://0.0.0.0:{{ .Values.config.listenPort }}
```

Comment:
Please check if this affects `livenessProbe` as well. It's hardcoded to `8000` in `deployment.yaml`.